### DISCUSSION TITLE
UDEMY öğrencilerine deprem.io duyurusunun yapılması

### DISCUSSION THREAD

**Discussion by s3audi** [acikkaynak/deprem-yardim-projesi#88] [Feedback]
50binden fazla öğrencisi olan gruplarda deprem.io duyurusu yapılıyor.
Siz de paylaşabilirsiniz. Hızlı bir tanınırlık çalışması olur.

**Comment by ramazansancar** [comment]
Merhaba,
Fikriniz için teşekkürler. Sosyal medya ekibimize iletiyorum.

**Comment by Batuhandgenc** [comment]
Konuyla ilgileneceğiz
